packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -6,7 +6,7 @@
 
 import { FluidObject } from '@fluidframework/core-interfaces';
 import { IChannel } from '@fluidframework/datastore-definitions';
-import type { IErrorBase } from '@fluidframework/core-interfaces';
+import { IErrorBase } from '@fluidframework/core-interfaces';
```

Comment:
Expected:  I verified that loss of 'type' is the result of importing from -public.d.ts.  It's not related to the package.json changes below.